bin/evm2wasm.js

```diff
@@ -1,17 +1,75 @@
 #!/usr/bin/env node
 
 const evm2wasm = require('../index.js')
-const ethUtil = require('ethereumjs-util')
-
-const input = ethUtil.toBuffer(process.argv[2])
-
-evm2wasm.evm2wasm(input, {
-  stackTrace: process.argv[3] === 'trace',
-  tempName: 'temp',
-  inlineOps: true,
-  wabt: true
-}).then(function (output) {
-  console.log(output.toString('binary'))
-}).catch(function (err) {
-  console.error('Failed: ' + err)
-})
+const argv = require('minimist')(process.argv.slice(2))
+const fs = require('fs')
+
+// convert evm bytecode to WASM or WAST
+function convert (bytecode, wast) {
+  return new Promise((resolve, reject) => {
```

Comment:
Not entirely sure why this had to be a promise? :)

Reply:
Because it calls a function that is async?